package.json

```diff
@@ -31,13 +31,8 @@
         "build": "react-scripts build",
         "test": "react-scripts test --watchAll=false",
         "eject": "react-scripts eject",
-        "precommit": "lint-staged --concurrent false"
-    },
-    "eslintConfig": {
-        "extends": [
-            "react-app",
-            "react-app/jest"
-        ]
+        "precommit": "lint-staged --concurrent false",
+        "prepare": "husky install"
```

Comment:
Can you add a section to readme for running this command when setting up the project?

Reply:
Actually NABD for now. You can do it if you want, I'll approve the PR

Reply:
Prepare is an npm lifecycle, it'll be run automatically when you do `npm ci` or `npm install`. Please refer to the documentation here https://docs.npmjs.com/cli/v8/using-npm/scripts#pre--post-scripts

So whenever user downloads packagaes, husky will be automatically installed and initialized!😁

Reply:
Also you can refer to [Husky documentation](https://typicode.github.io/husky/#/)
![image](https://user-images.githubusercontent.com/61606375/190023838-62329eb4-67b7-46ee-b33e-eeb530ccfed6.png)

Reply:
Nice!